File: 프로그래머스/0/120837. 개미 군단/개미 군단.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int hp) {
    int answer = 0;
    int a =0,b=0,c=0;
    
    a = hp/5;
    hp %= 5;
    b = hp/3;
    hp %= 3;
    c = hp;
        
    
    answer = a+ b+c;
    
    
    
    return answer;
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.py
def solution(hp):
    answer = 0
    
    
    
    if hp >= 5:
        answer+= int(hp//5)
        hp%=5
    if hp >= 3:
        answer+= int(hp//3)
        hp%=3
    
    answer+= hp
    
    
    
    return answer